packages/app/src/routes/routes.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  AccountPage,
+  CasePage,
+  CasesPage,
+  EditAccountPage,
+  FormsPage,
+  NotificationsPage,
+  OverviewPage,
+  TasksPage,
+  TaskPage,
+  ThemesPage,
+  FormPage,
+} from "@nl-portal/nl-portal-user-interface";
+import { paths } from "../constants/paths";
+import { config } from "../constants/config";
+import { Navigate } from "react-router-dom";
+
+export const routes = [
+  {
+    path: "*",
+    element: <Navigate to={sessionStorage.getItem("entryUrl") || "/"} />,
+  },
+  {
+    path: paths.overview,
+    element: (
+      <OverviewPage
+        showIntro="true"
+        openFormsFormId={config.OPEN_FORMS_FORM_ID}
+        showCasesPreview
+        casesPreviewLength={6}
+      />
+    ),
+  },
+  {
+    path: paths.cases,
+    children: [
+      {
+        index: true,
+        element: <CasesPage />,
+      },
+      {
+        path: paths.case(),
+        element: <CasePage showContactTimeline />,
+      },
+    ],
+  },
+  {
+    path: paths.tasks,
+    children: [
+      {
+        index: true,
+        element: <TasksPage />,
+      },
+      {
+        path: paths.task(),
+        element: <TaskPage />,
+      },
+    ],
+  },
+  {
+    path: paths.notifications,
+    element: <NotificationsPage />,
+  },
+  {
+    path: paths.themes,
+    element: <ThemesPage />,
+  },
+  {
+    path: paths.forms,
```

Comment:
Deze zijn verwijderd in deze MR: https://github.com/nl-portal/nl-portal-frontend-libraries/pull/122

Misschien als je merge conflicts gaat oplossen dat je hier wel wat issues mee krijgt.

Reply:
Master is terug gemerged in deze feature en conflicten zijn opgelost.